code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -1,5 +1,5 @@
 //The chests dropped by mob spawner tendrils. Also contains associated loot.
-
+GLOBAL_LIST_EMPTY(bloodmen_list)
```

Comment:
I THINK this can be a var/static/list inside bloodmen themselves rather than a global variable

Reply:
the global variable bit was because of the spell that would directly create a bloodman without any bloodlings, along with the other spell that would summon them all to the caster. When I described my idea to another coder, I was told to use a global list.

Reply:
static list could work too, if theos doesn't want another global. I can show you how to use it (I was the one that suggested a global)